### ISSUE TITLE
Comma before "Read More" for Deutch

### ISSUE THREAD

**Issue by ValentinaPC** [mozilla/addons-frontend#2597]
Steps to reproduce:
1. Load details page for an add-on that has a long description i.e. Lightbeam for Firefox
2. Change the language to "de" : https://addons.allizom.org/de/firefox/addon/lightbeam/?src=hp-dl-featured
3. Observe "Read More" link

Expected results:
There are no display or layout issues.

Actual results:
A  comma is displayed before "Read more" link.

Notes/Issues:
Reproduced in all AMO servers on desktop and device.
Verified on AMO-prod FF53(Win7, Android 7.1.2)
Screenshot for this issue:
![2017-06-16_1203](https://user-images.githubusercontent.com/15685960/27219897-eb344b00-528b-11e7-95aa-80b2e1aaad66.png)

**Comment by muffinresearch** [comment]
This is an issue in the translation. https://github.com/mozilla/addons-frontend/blob/master/locale/de/LC_MESSAGES/amo.po#L613

The span is hidden but the translation is adding a comma after the hidden part as if it's visible.

**Comment by tsl143** [comment]
Just to confirm, that is dealt by Pontoon, we dont change in amo.po files, right?

**Comment by muffinresearch** [comment]
@koehlermichael hi would it be possible to update the localization and remove the ',' after the hidden content?

**Comment by muffinresearch** [comment]
This also relates to https://github.com/mozilla/addons-frontend/issues/1727

**Comment by koehlermichael** [comment]
I can remove anything but if I do remove the comma the wording doesn’t make sense anymore.

I can rephrase it to say „Expand. Read more.“ thus removing the link between both segments, if that makes sense in that context.

**Comment by muffinresearch** [comment]
> I can rephrase it to say „Expand. Read more.“ thus removing the link between both segments, if that makes sense in that context.

Yes that sounds good. We'll be changing the text in due course since from an accessibility perspective the hidden text isn't adding enough context as detailed in #1727

**Comment by koehlermichael** [comment]
There should be an update to the relevant file just now. Let’s see if this works as needed.

**Comment by muffinresearch** [comment]
Looks good to me.

**Comment by ValentinaPC** [comment]
Verified as fixed on AMO-dev FF55(Win 7, Android 7.2.1).
Postfix screenshot:
![readmore-de](https://user-images.githubusercontent.com/15685960/29970747-74ea24e4-8f2e-11e7-90bd-9afb0686866a.png)
